README.md

```diff
@@ -19,6 +20,10 @@ deployment scheme.
 technology!](https://drive.google.com/file/d/0B667S2caJiy7QVpzVVFNQVdyaVE/view?usp=sharing) <<<**
 
 
+
+
```

Comment:
Do we need space? I do not see any difference though. 

View : https://github.com/ramkrsna/gluster-kubernetes/tree/addDocs